engineio/src/socket.rs

```diff
@@ -117,7 +117,7 @@ impl Socket {
     }
 
     pub fn close(&self) -> Result<()> {
-        self.emit(Packet::new(PacketId::Close, Bytes::from_static(b"")), false)?;
+        self.emit(Packet::new(PacketId::Close, Bytes::from_static(b"")))?;
```

Comment:
I think `Bytes::default()` or `Bytes::new()` is better here. The [documentation](https://docs.rs/bytes/1.1.0/bytes/struct.Bytes.html#method.new) here also mentions that empty bytes also don't allocate.

Reply:
This involves all occurrences of a `Bytes::from_static(b"")` call.

Reply:
17a394c9339f537772b689814feb091a487ee34d